mirrormanager2/templates/fedora/master_noauth.html

```diff
@@ -5,47 +5,52 @@
     <title>{% block title %}{% endblock %} - MirrorManager</title>
     <link rel="shortcut icon" type="image/vnd.microsoft.icon"
         href="{{ url_for('static', filename='favicon.ico')}}"/>
-    <link rel="stylesheet" type="text/css" media="screen"
-        href="{{ url_for('static', filename='koji.css') }}"/>
+        <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-1.0.2/fedora-bootstrap.css"
+              type="text/css" rel="stylesheet" />
     <link rel="stylesheet" type="text/css" media="screen"
         href="{{ url_for('static', filename='mirrormanager2.css') }}"/>
+    <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/hack.css" type="text/css" rel="stylesheet" />
+    <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
+    <link href="https://ryanlerch.fedorapeople.org/font-awesome.css" type="text/css" rel="stylesheet" />
```

Comment:
yeah, i'll change it back, sorry!

Just waiting on a patch that has been sitting in the infra queue for a while that makes this font work for local testing too.